client/src/components/Products.js

```diff
@@ -18,7 +18,21 @@ export default function Products() {
 
     const getProducts = async () => {
         const endpoint = `products`
-        await handleGet(endpoint, setProducts)
+        const host = process.env.REACT_APP_NODE_LOCAL || process.env.REACT_APP_NODE_PROD
+        const url = `${host}/${endpoint}`
```

Comment:
so the solution was to use these slanted single quotes?

Reply:
Do you mean for the connections issue we had? I think the issue we had during our call was that we forgot to restart the server when we changed the .env prop values. I had completely forgotten that React requires a server restart if env variables are changed, which might be why it didn't work for you. My mistake. The solution for the list order changing (on both the products page and the shopping cart page) was to sort the array using a place_in_catalog (products) or place_in_cart (cart_items).